apps/src/craft/craft.js

```diff
@@ -281,6 +281,10 @@ Craft.init = function (config) {
       if (!config.level.showPopupOnLoad) {
         Craft.initializeAppLevel(config.level);
       }
+
+      // Adjust visualizationColumn width.
+      var visualizationColumn = document.getElementById('visualizationColumn');
+      visualizationColumn.style.width = this.nativeVizWidth + 'px';
```

Comment:
Would this stop the CSS browser-window-dimension-based sizing from happening pre-handle-grab? e.g. open page, drag window size up and down, should change size based on browser window size